airflow/models/taskinstance.py

```diff
@@ -1255,11 +1258,13 @@ def signal_handler(signum, frame):  # pylint: disable=unused-argument
 
         end_time = time.time()
         duration = timedelta(seconds=end_time - start_time)
+        dag_id = task_copy.dag_id
+        task_id = task_copy.task_id
         Stats.timing(
-            f'dag.{task_copy.dag_id}.{task_copy.task_id}.duration',
-            duration,
+            'dag.{dag_id}.{task_id}.duration', duration, labels={"dag_id": dag_id, "task_id": task_id}
         )
-        Stats.incr(f'operator_successes_{self.task.task_type}', 1, 1)
+        task_type = self.task.task_type
+        Stats.incr('operator_successes_{task_type}', 1, 1, labels={"task_type": task_type})
         Stats.incr('ti_successes')
```

Comment:
Would it be possible to add the dag/task tags to ti_successes and ti_failures as well? I think this would useful for aggregation so we can ignore or watch specific dags for failures